runtime.md

```diff
@@ -121,11 +121,18 @@ When the process in the container is stopped, irrespective of it being as a resu
 This operation MUST generate an error if it is not provided the container ID.
 Attempting to delete a container that does not exist MUST generate an error.
 Attempting to delete a container whose process is still running MUST generate an error.
-Deleting a container MUST delete the resources that were created during the `create` step.
-Note that resources associated with the container, but not created by this container, MUST NOT be deleted.
 Once a container is deleted its ID MAY be used by a subsequent container.
 
+Deleting a container MUST delete the resources that were created during the `create` step.
+Resources associated with the container, but not created by this container, MUST NOT be deleted.
+
+On Linux, [`mounts`](config.md#mounts), [`root`](config.md#root), [`linux.devices`](config-linux.md#devices), and other filesystem changes belong to the container which created the [container mount namespace][container-namespace3].
+If a container joins an existing mount namespace and applies those settings, deleting the container MUST NOT reverted the settings.
+Similarly [`hostname`](config.md#hostname) changes belong to the container which created the container UTS namespace.
+[`linux.resources`](config-linux.md#control-groups) settings belong to the container which created the associated cgroup.
```

Comment:
```
Deleting a container MUST delete the resources that were created during the `create` step.
Resources associated with the container, but not created by this container, MUST NOT be deleted.
```

For ownership concept, it is reasonably enough for me.

I think its not harmful to have more examples to illustrate the point, but they need to be clearly spelled out as example so they wont be mistook as the only cases that we will support.

WDYT about something like:

```
In the following example, when the container joins an existing mount namespace on Linux and changes root with the following config.json

{
root: /new/root,
linux: { namespaces: [ {type: mount, path: /proc/1234/ns/mnt} ] },
}

deleting the container MUST not remove the existing mount namespace, and MUST not undo the changes to root

In the following example, when the container joins an existing UTS namespace on Linux and changes hostname with the following config.json

{
hostname: runtimespec,
linux: { namespaes: [{type: uts, path: /proc/1234/ns/uts} ] }
}

deleting the container MUST not remove the existing UTS namespace, and MUST not undo the changes to hostname.
```